### DISCUSSION TITLE
BIRT. Visiblity For specific outputs. Missing XLS.

### DISCUSSION THREAD

**Discussion by mrEugenS** [eclipse-birt/birt#1136] [Q&A]
The list of export formats does not contain xls when controlling the visibility of elements.
Is it possible to patch the list in the IDE?
![eclipse_xls](https://user-images.githubusercontent.com/91918109/208477365-bd7f5423-f490-4ac3-8ca5-b2e463d7326f.png)

**Comment by wimjongman** [comment]
![image](https://user-images.githubusercontent.com/180969/208489347-5d3fea39-997a-415b-af2c-21bd47491547.png)

**Reply by mrEugenS** [reply]
No. These are separate formats.
If you add code to the template with your hands, in this case, the export goes as it should. But in this case, falls into an Eclipse error.

```
<structure>
<property name="format">xls</property>
<expression name="valueExpr" type="javascript">true</expression>
</structure>
```

![eclipse_xls2](https://user-images.githubusercontent.com/91918109/208498167-2900761e-33cc-44a7-8e6c-efda7b2de4ac.png)

**Reply by speckyspooky** [reply]
The old XLS-emitter was scoped out since BIRT version 4.8.
The primary excel-emitter of XLS & XLSX are the both version from the Spudsoft-Excel-Emitter.

If you use one of the latest version your will see that BIRT have reduced the excel-options
and provide the mentioned excel-formats.
